pom.xml

```diff
@@ -67,22 +67,23 @@
     <!-- dependencies -->
     <!-- latest version from apache pulsar -->
     <pulsar.version>2.10.2</pulsar.version>
-    <scala.version>2.12.17</scala.version>
+    <scala.version>2.12.10</scala.version>
```

Comment:
It downgrades some dependencies; will this have some effects?

Reply:
It should be okay, we can revisit this change once the admin PRs are done.